i/ibm-namespace-scope-operator/ibm-namespace-scope-operator_ubi_8.5.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : IBM/ibm-namespace-scope-operator
+# Version       : v1.5.0, v1.0.1
+# Source repo   : https://github.com/IBM/ibm-namespace-scope-operator
+# Tested on     : ubi 8.5
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Ambuj kumar <Ambuj.Kumar3@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_PATH=github.com/IBM/
+PACKAGE_NAME=ibm-namespace-scope-operator
+PACKAGE_VERSION=${1:-v1.5.0}
+PACKAGE_URL=https://github.com/IBM/ibm-namespace-scope-operator
+
+# Install dependencies
+yum install -y make git wget gcc
+
+# Install unit test dependencies and run unit tests
+#rm -rf $CWD/kubebuilder-tools
```

Comment:
Please remove code that is not required.
The variable `CWD` is used without initializing it.

Reply:
Hi prashant, done the changes according to review comment